### PR TITLE
Adding sample asset populator.

### PR DESCRIPTION
**Pull request by Rajith90**
This sample can be used to populate wsdls, wadls and schemas for G-Reg, and corresponding rest services and soap services will be created.

### CODE REVIEW

File: modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+public class Main {
+    private static String cookie;
+    private static final String username = "admin";
+    private static final String password = "admin";
+    private static final String serverURL = "https://localhost:9443/services/";
+
+    private static void setSystemProperties() {
+        String trustStore = System.getProperty("carbon.home") + File.separator + "repository" + File.separator +
+                "resources" + File.separator + "security" + File.separator + "wso2carbon.jks";
```
Comment (cnapagoda) [inline modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java]: Can't we use StringBulder here?

File: modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	private static void addWsdlGar(ResourceServiceClient resourceServiceClient, String projectPath) throws Exception {       
+        DataHandler dh = new DataHandler(new URL("file://"+projectPath+"/wsdl_new.gar"));
+        resourceServiceClient.addResource("/_system/governance/trunk/test/wsdls/",
+                                               "application/vnd.wso2.governance-archive", null, dh, null,null);
+    }
+
+	private static void addSchemaGar(ResourceServiceClient resourceServiceClient, String projectPath) throws Exception {
+	DataHandler dh = new DataHandler(new URL("file://"+projectPath+"/schemas.gar"));
+        resourceServiceClient.addResource("/_system/governance/trunk/test/schemas/",
+                                               "application/vnd.wso2.governance-archive", null, dh, null,null);
```
Comment (cnapagoda) [inline modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java]: I think, you can move "application/vnd.wso2.governance-archive" into variable

Comment (Rajith90) [inline modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java]: PR sent : https://github.com/wso2/product-greg/pull/367/
Thanks!

File: modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java (hunk 1 above)
Comment (Rajith90) [inline modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java]: PR sent : #367
Thanks!

File: modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+import javax.activation.DataHandler;
+import javax.activation.DataSource;
+import java.net.URI;
+import java.net.URL;
+import javax.xml.namespace.QName;
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.*;
```
Comment (Rajith90) [inline modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java]: Optimized imports : https://github.com/wso2/product-greg/pull/300/

File: modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        System.setProperty("carbon.repo.write.mode", "true");
+    }
+
+    public static void main(String[] args) {
+        try {
+
+            setSystemProperties();
+
+            String axis2Configuration = System.getProperty("carbon.home") + File.separator + "repository" +
+                    File.separator + "conf" + File.separator + "axis2" + File.separator + "axis2_client.xml";
```
Comment (Rajith90) [inline modules/samples/data-populator/Populator/src/org/wso2/carbon/registry/samples/populator/Main.java]: PR sent : #367
Thanks!

**Pull request merged**